### PR TITLE
Add script to check SDL does not resend OnSeekMediaClockTimer if seek…

### PR DESCRIPTION
**Pull request by OVikhrov**
…Time seconds out upper bound
@aarotar Please review

### CODE REVIEW

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Out_Upper_Bound.lua [hunk 1]
```diff
@@ -0,0 +1,94 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [MOBILE_API] New OnSeekMediaClockTimer notification
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Out_Upper_Bound.lua]: @OVikhrov, please update Requirement summary with related **functional** requirement also

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Out_Upper_Bound.lua [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+function Test:Preconditions_ActivateApp()
+commonSteps:ActivateAppInSpecificLevel(self, self.applications["Test Application"])
+EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_OnSeekMediaClockTimer_secnds_Value_Of_seekTime_Is_Out_Upper_Bound()
+ self.hmiConnection:SendNotification("UI.OnSeekMediaClockTimer",{seekTime =  {hours = 30, minutes = 30, seconds = 60}, appID = self.applications["Test Application"]})
+
```
Comment (aarotar) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Out_Upper_Bound.lua]: @OVikhrov, please check syntax in script

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Out_Upper_Bound.lua (hunk 1 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Out_Upper_Bound.lua]: Accepted

File: test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Out_Upper_Bound.lua (hunk 2 above)
Comment (OVikhrov) [inline test_scripts/API/OnSeekMediaClockTimer/ATF_N_OnSeekMediaClockTimer_seekTime_seconds_Out_Upper_Bound.lua]: Accepted

**Comment by OVikhrov** [comment]
@aarotar Plese review iteration#2

**Pull request merged**